embedding/mbedk64f/js2c.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
```

Comment:
Yes with some file path difference. This seems to be done with making common js example code.